server/templates/general/index.html

```diff
@@ -2,8 +2,39 @@
 
 {% block head %}
 {{ super() }}
-<script src="/js/index/tab_headers.js"></script>
-<script src="/js/index/modal_box_index_page.js"></script>
+    <script src="/js/index/home_page_processing.js"></script>
+    <script src="/js/index/home_page_preloader.js"></script>
+    <script src="/js/index/home_page_table_tab.js"></script>
+    <script src="/js/index/home_page_modal_windows.js"></script>
+    <script>
+    "use strict";
+    /**
+     * Callback function, called on success, for reading user roles data
+     * @param data      - JSON object
+     */
+    function showUserRoles (data) {
+       for (let index = 0; index < data.userRoles.length ; index++) {
+            drawCheckboxRolesModalWindow(data.userRoles[index]);
+       }
+    }
+
+    /**
+     * The function of drawing checkboxes proceeding
+     * @param indexRole     - JSON object (format data.userRoles[index])
+     */
+    function drawCheckboxRolesModalWindow (indexRole) {
+        const CHECKBOX_CONTAINER = '#modal_user_checkbox_roles';
+        let $checkbox = $("<div class='checkbox-container'>");
+
+        $(CHECKBOX_CONTAINER).append($checkbox);
+        $checkbox.append($('<input class= "checkbox modal-user-checkbox" type="checkbox">').val(indexRole.name));
+        $checkbox.append($('<label for="checkbox"></label>').text(indexRole.description));
+    }
+
+    sendJSONRequest("/home/getUserRolesList", null, RequestMethod.GET, beforeSendEmpty,
```

Comment:
Request should be executed after the page is completely loaded.
Consider wrapping it into document.ready listener.